datadog-opentelemetry/src/lib.rs

```diff
@@ -477,3 +513,54 @@ pub fn make_test_tracer(shared_config: Arc<Config>) -> (SdkTracerProvider, Datad
         None,
     )
 }
+
+use opentelemetry_sdk::metrics::SdkMeterProvider;
+
+/// Builder for Datadog Metrics with OTLP transport
+pub struct DatadogMetricsBuilder {
+    config: Option<Config>,
+    resource: Option<Resource>,
+    export_interval: Option<std::time::Duration>,
+}
+
+impl DatadogMetricsBuilder {
+    pub fn with_config(mut self, config: Config) -> Self {
+        self.config = Some(config);
+        self
+    }
+
+    pub fn with_resource(mut self, resource: Resource) -> Self {
+        self.resource = Some(resource);
+        self
+    }
+
+    pub fn with_export_interval(mut self, interval: std::time::Duration) -> Self {
+        self.export_interval = Some(interval);
+        self
+    }
+
+    pub fn init(self) -> Result<SdkMeterProvider, String> {
+        let (meter_provider, _) = self.init_local()?;
+        opentelemetry::global::set_meter_provider(meter_provider.clone());
+        Ok(meter_provider)
+    }
+
+    pub fn init_local(self) -> Result<(SdkMeterProvider, ()), String> {
```

Comment:
Is the tuple needed?
```suggestion
    pub fn init_local(self) -> Result<SdkMeterProvider, String> {
```